sql/src/main/java/io/crate/expression/reference/sys/job/JobContext.java

```diff
@@ -23,27 +23,20 @@
 
 import io.crate.auth.user.User;
 
-import javax.annotation.Nullable;
 import java.util.UUID;
 
 public class JobContext {
 
     public final UUID id;
-    @Nullable
     public final String username;
     public final String stmt;
     public final long started;
 
-    public JobContext(UUID id, String stmt, long started, @Nullable User user) {
+    public JobContext(UUID id, String stmt, long started, User user) {
```

Comment:
> would it make sense to receive the username directly instead of the user?

We could, but I'd see that as separate change and right now it's not too much of a hassle to pass in the whole user.